backend/tracim_backend/lib/core/mention.py

```diff
@@ -0,0 +1,136 @@
+import typing
+
+from bs4 import BeautifulSoup
+from bs4 import Tag
+from pluggy import PluginManager
+
+from tracim_backend.config import CFG
+from tracim_backend.lib.core.content import ContentApi
+from tracim_backend.lib.core.event import BaseLiveMessageBuilder
+from tracim_backend.lib.core.event import EventApi
+from tracim_backend.lib.core.plugins import hookimpl
+from tracim_backend.lib.core.workspace import WorkspaceApi
+from tracim_backend.lib.utils.request import TracimContext
+from tracim_backend.models.data import Content
+from tracim_backend.models.event import EntityType
+from tracim_backend.models.event import OperationType
+from tracim_backend.models.event import Event
+from tracim_backend.models.tracim_session import TracimSession
+
+
+class Mention:
+    """A mention with its attributes: id and recipient."""
+
+    def __init__(self, recipient: str, id_: str) -> None:
+        self.recipient = recipient
+        self.id = id_
+
+    def __eq__(self, other) -> bool:
+        return (
+            isinstance(other, Mention) and other.recipient == self.recipient and other.id == self.id
+        )
+
```

Comment:
It seems this could be simplified by using named tuples, or dataclasses

- https://docs.python.org/3.8/library/typing.html#typing.NamedTuple
- https://docs.python.org/3.8/library/collections.html#collections.namedtuple
- https://docs.python.org/3/library/dataclasses.html

With these, you won't have to write these `init` and `_eq__` method

(Compatibility with Python 3.5 or 3.6 might be an issue though)

Reply:
Yes, dataclasses are not possible with Python 3.5 (there is a backport for 3.6). Could use named tuple it just doesn't play well with typing (I do not think it is possible to type the arguments to namedtuple, at least for 3.5: https://docs.python.org/3/library/typing.html#typing.NamedTuple).

Reply:
I am also for dropping Python 3.5, I think this is holding us back in this kind of situation. Python 3.7 can still be installed on Stretch using the backports.

This discussion can be resolved.

Reply:
> Note: I would be in favor of dropping 3.5 support, but this is a general discussion as this would likely lead to also drop stretch support…

This is a product strategic decision, not a "ease of development decision" (ie to be discussed more largely)

Reply:
Sure, I think this is what @grignards meant by saying "general discussion".